ui/src/main/webapp/wizards/ldap/stages/network-settings.js

```diff
@@ -18,6 +18,7 @@ import {
 } from 'admin-wizard/inputs'
 
 import { groupErrors } from './errors'
+import {getFriendlyMessage} from '../../../lib/graphql-errors'
```

Comment:
Things in lib can be referenced directly. You can reduce the import to graphql-errors.